Wireframe/index.html

```diff
@@ -10,23 +10,58 @@
     <header>
       <h1>Wireframe</h1>
       <p>
-        This is the default, provided code and no changes have been made yet.
+        Let's explore three  concepts.
       </p>
     </header>
     <main>
+
+
       <article>
         <img src="placeholder.svg" alt="" />
-        <h2>Title</h2>
+        <h2> What is the purpose of a README file?</h2>
         <p>
```

Comment:
If you review the exercise requirements, you'll see that the wireframe has the title description centered on the page. As much as possible, try to always create your webpage as similar to the wireframe as possible!